locales/es/es.json

```diff
@@ -1,48 +1,48 @@
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Si tiene problemas para hacer clic en el botón \":actionText\", copie y pegue la siguiente URL \nen su navegador web:",
     "Invalid signature.": "Firma no válida.",
-    "Log out": "Finalizar sesión",
-    "Logout Other Browser Sessions": "Cerrar las demás sesiones",
-    "Manage and logout your active sessions on other browsers and devices.": "Administre y cierre sus sesiones activas en otros navegadores y dispositivos.",
-    "Nevermind": "Olvidar",
+    "Login": "Iniciar sesión",
+    "Logout": "Finalizar sesión",
+    "Name": "Nombre",
     "Not Found": "No encontrado",
     "Oh no": "Oh no",
     "Page Expired": "Página expirada",
-    "Pagination Navigation": "Navegación por los enlaces de paginación",
+    "Password": "Contraseña",
     "Please click the button below to verify your email address.": "Por favor, haga clic en el botón de abajo para verificar su dirección de correo electrónico.",
     "Please confirm your password before continuing.": "Por favor confirme su contraseña antes de continuar.",
-    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Por favor ingrese su contraseña para confirmar que desea cerrar las demás sesiones de otros navegadores en todos sus dispositivos.",
-    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor proporcione la dirección de correo electrónico de la persona que le gustaría agregar a este equipo. La dirección de correo electrónico debe estar asociada a una cuenta existente.",
     "Regards": "Saludos",
-    "results": "resultados",
+    "Register": "Registrarse",
+    "Remember Me": "Mantener sesión activa",
+    "Reset Password": "Restablecer contraseña",
+    "Reset Password Notification": "Notificación de restablecimiento de contraseña",
+    "Send Password Reset Link": "Enviar enlace para restablecer la contraseña",
     "Server Error": "Error del servidor",
     "Service Unavailable": "Servicio no disponible",
-    "Showing": "Mostrando",
-    "The :attribute must contain at least one letter.": "La :attribute debe contener al menos una letra.",
-    "The :attribute must contain at least one number.": "La :attribute debe contener al menos un número.",
-    "The :attribute must contain at least one symbol.": "La :attribute debe contener al menos un símbolo.",
-    "The :attribute must contain at least one uppercase and one lowercase letter.": "La :attribute debe contener al menos una letra mayúscula y una minúscula.",
-    "The given :attribute has appeared in a data leak. Please choose a different :attribute.": "La :attribute proporcionada se ha visto comprometida en una filtración de datos (data leak). Elija una :attribute diferente.",
```

Comment:
Would the new translations belonging to the new Password Validation rule be lost?

Reply:
The translation keys contained in the Laravel kernel files are really very few and I think they were all in the previous versions. I think to leave them and if a new one or a surplus arises, then a PR is proposed to add or remove it as it had always been done.

Reply:
I have updated the `todo` in the PR description. Do you think need to add something else to it?